e2e-tests/tests/add-entity-page.spec.ts

```diff
@@ -11,11 +11,15 @@ studioTest("can add an entity page", async ({ page, studioPage }) => {
   const pageInTree = page.getByText("EntityPage");
   await expect(pageInTree).toHaveCount(0);
 
+  const addPageSection = studioPage.addPageSection;
+  await addPageSection.selectPageType(true);
+  await addPageSection.enterStreamScope({}, true);
+  await addPageSection.enterBasicPageData("EntityPage", "entity-page");
```

Comment:
The basic page data modal is a little different between static and entity pages. For entity pages, the URL slug field should be populated with `[[slug]]`. Do we want to add a screenshot for that?